include/nds/arm9/math.h

```diff
@@ -129,7 +129,13 @@ static inline int32_t sqrtf32_result(void)
 }
 
 #pragma GCC diagnostic push
+// GCC does not recognize clang diagnostic pragma, ignore it here
```

Comment:
Wouldn't it be better to do ``#ifdef __clang__`` or something like that?

Reply:
Yea, not sure why I didn't think of that.